velox/functions/prestosql/tests/ArrayMaxTest.cpp

```diff
@@ -145,11 +145,22 @@ TEST_F(ArrayMaxTest, longVarcharNoNulls) {
 
 // Test documented example.
 TEST_F(ArrayMaxTest, docs) {
-  auto input = makeNullableArrayVector<int32_t>(
+  auto input1 = makeNullableArrayVector<int32_t>(
```

Comment:
a1, a2, a3 variable names are an anti-pattern.

Since this is a simple function, no need to test it on vectors. Writing test on single values will make it easier to read.

```
ASSERT_EQ(3, arrayMax<int32_t>({1, 2, 3}));
...

```

Reply:
We can change the tests, ideally we should change all tests not just this one for array min/max. I can do it in the next PR-since its more refactoring that this one.

Reply:
Sounds good. Looking forward to a follow-up.